config.go

```diff
@@ -2,27 +2,40 @@ package bigqueue
 
 import (
 	"errors"
+	"math"
 	"os"
+	"time"
 )
 
 const (
 	cDefaultArenaSize = 128 * 1024 * 1024
 
 	// tail, head and a buffer arena, hence 3
 	cMinMaxInMemArenas = 3
+
+	// large default values signifying that we rely on Linux's pdflush thread
+	cFlushIntervalMutateOps = math.MaxInt64
+	cFlushElapsedDuration   = math.MaxInt64
 )
 
 var (
 	// ErrTooSmallArenaSize is returned when arena size is smaller than OS page size
 	ErrTooSmallArenaSize = errors.New("too small arena size")
 	// ErrTooFewInMemArenas is returned when number of arenas allowed in memory < 3
 	ErrTooFewInMemArenas = errors.New("too few in memory arenas")
+	// ErrFlushIntervalMutateOpsNotGreaterThanZero is returned when number of mutate operations
+	// set as flush interval is not greater than 0
+	ErrFlushIntervalMutateOpsNotGreaterThanZero = errors.New("number of mutate operations as flush interval must be greater than zero ")
```

Comment:
I think you could just use the same error in both cases, say`ErrGreaterThanZero`. Both values have to be positive.